releasing/plugin-commands-deploy/test/deploy.test.ts

```diff
@@ -213,3 +213,53 @@ test('deploy with dedupePeerDependents=true ignores the value of dedupePeerDepen
   project.has('is-positive')
   expect(fs.existsSync('sub-dir/deploy')).toBe(false)
 })
+
+// Regression test for https://github.com/pnpm/pnpm/issues/8297 (pnpm deploy doesn't replace catalog: protocol)
+test('deploy works when workspace packages use catalog protocol', async () => {
```

Comment:
Although most tests are in `pkg-manager/core`, I think this test has to be in `releasing/plugin-commands-deploy` to use the `deploy` function.